### PR TITLE
DEVXP-2811 - Opensource

### PR DESCRIPTION
**Pull request by vschoener**
This PR updates the Readme to add some opensource details, License and how to contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,17 @@ https://libyear.com/
 
 ![Demo Image](./docs/demo.png)
 
-## How to install
-`pip install libyear`
+## Description
 
+This project is used for python project (pip and poetry) to measure what is your libyear dependencies.
```
Comment (vitorquintanilha) [inline README.md]: For poetry you need to generate a `requirements.txt` file with a command similar to `poetry export -f requirements.txt --output requirements.txt`.

Also, we I think "measure what is your libyear dependencies" it maybe not that clear. "calculate the libyear metric for your dependencies" might be bit easier to understand.

File: README.md [hunk 2]
```diff
@@ -69,6 +80,243 @@ libyear -r requirements.txt
...
+      "latest_version": "1.16.0",
+      "libyear": "2.4"
+    },
+    {
+      "name": "flake8-bugbear",
+      "version": "19.3.0",
+      "latest_version": "24.2.6",
+      "libyear": "4.87"
+    },
+    {
```
Comment (vitorquintanilha) [inline README.md]: We could probably leave just 3 items on the array so it is more readable ^^

File: README.md [hunk 3]
```diff
@@ -86,3 +334,24 @@ Apps below 10 libyears are considered to be healthy apps. We regularly rescue pr
 
 ## References
 J. Cox, E. Bouwers, M. van Eekelen and J. Visser, Measuring Dependency Freshness in Software Systems. In Proceedings of the 37th International Conference on Software Engineering (ICSE 2015), May 2015 https://ericbouwers.github.io/papers/icse15.pdf
+
+
+## Contributing
+
+If you want to contribute to this project, please fork it, create a dedicated branch to make your changed and simply raise a pull request that targets this repository.
+
+## License
```
Comment (vitorquintanilha) [inline README.md:140]: I think we might need to keep the MIT license from the original repo: https://github.com/nasirhjafri/libyear/blob/master/setup.py#L26

File: README.md (hunk 2 above)
Comment (vschoener) [reply README.md]: Yeah you right :)

File: README.md (hunk 3 above)
Comment (vschoener) [reply README.md:140]: Indeed, I have put MIT text grabbed from https://opensource.org/license/mit

**Review by vitorquintanilha — COMMENTED** [review]
Just to be fully compliant w/ python standards, we could add those to `pyproject.toml` ([ref1](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license), [ref2](https://python-poetry.org/docs/pyproject/#license))

```
[tool.poetry]
# ....
license = "MIT"

[project]
license = {text = "MIT License"}
```

**Review by vitorquintanilha — APPROVED** [review]

**Pull request merged**